### PR TITLE
feat(intro): init faq - I297

### PR DESCRIPTION
**Pull request by tbrooke**
# Issue #<NUMBER HERE>
<OVERALL SUMMARY OF PULL REQUEST>

### Changes
- <INSERT DESCRIPTION OF CHANGES>
  - <SUB NOTES OF DESCRIPTIONS>
- <SUMMARIZE ALL COMMITS IN PULL REQUEST>

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #<NUMBER HERE>
- Pull Request #<NUMBER HERE>

### CODE REVIEW

File: docs/accordproject-faq.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+### What is the Accord Project and what is its purpose?
+
+The Accord Project is a non-profit, member-driven organization that builds open source code and documentation for smart legal contracts for use by transactional attorneys, business and finance professionals, and other contract users. Open source means that anyone can use and contribute to the code and documentation and use it in their own software applications and systems free of charge.
+
+The purpose of the Accord Project is to establish and maintain a common and consistent legal and technical foundation for smart legal contracts. The Accord Project is organized into working groups focused on transactions: supply chain, financial services, intellectual property, venture and token sales, real estate and construction; and also on dispute resolution. The transactional working groups are assisted by the Technology Working Group, which builds the underlying open source code and specifications to codify the knowledge of the transactional working groups. More details about the internal governance of the Accord Project are available here.
```
Comment (irmerk) [inline docs/accordproject-faq.md:44]: Is there a link missing at the end of this paragraph?

File: docs/accordproject-faq.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+### How can I get involved?
+
+The working groups have calls at least once a month and use the Accord Project’s Slack group chat application (join by clicking here) for discussion. The dates, dial-in instructions, and agendas for the working groups are all listed in the Project’s public calendar and typically also in working group’s respective slack channels.
```
Comment (irmerk) [inline docs/accordproject-faq.md:49]: I think `The working groups have calls at least...` should be rephrased to not rely on the reader to have just read the previous FAQ.

File: docs/accordproject-faq.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+A primary purpose of the working groups is to develop a universally accessible and widely used open source library of modular, smart legal contract and smart clause templates and models that reflect input from transactional attorneys and other experts. Smart legal contract templates are built according to the Project’s Cicero Specification.
```
Comment (irmerk) [inline docs/accordproject-faq.md:51]: Could provide a link for `... Accord Project’s Cicero Specification`

File: docs/accordproject-faq.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Members are expected to at least provide some feedback into the templates and models relevant to a particular working group. You can immediately get started contributing smart legal contract templates and models by using the Accord Project’s Template Studio.
```
Comment (irmerk) [inline docs/accordproject-faq.md:53]: Could provide a link for `... Accord Project’s Template Studio`

File: docs/accordproject-faq.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Another purpose of the working groups is to use and help develop an easy-to-use programming language for building and executing smart legal contracts called Ergo. The goals of Ergo are to be accessible and usable by non-technical professionals, portable across, and compatible with, a variety of environments such as SaaS platforms and different blockchains, and meeting security, safety, and other requirements. You can use the Accord Project’s Template Studio to create and test your smart legal contract models and run them on other software systems.
```
Comment (irmerk) [inline docs/accordproject-faq.md:55]: Could provide a link for `... called Ergo.`

Unsure why but the second sentence seems awkward.

Last sentence could also include the VSCode Plugin in addition to Template Studio?

**Comment by tbrooke** [comment]
@jeromesimeon It evolved because I started rewriting the Faq and included sections for lawyers business developer - but I stepped back and basically just moved the existing faq but I still felt that lawyers should be promoted So I split out lawyers from the business section and tweakd it a little to get rid of some repetition

**Comment by jeromesimeon** [comment]
> @jeromesimeon It evolved because I started rewriting the Faq and included sections for lawyers business developer - but I stepped back and basically just moved the existing faq but I still felt that lawyers should be promoted So I split out lawyers from the business section and tweakd it a little to get rid of some repetition

oh I see what you're saying now. Thanks for the clarification. I don't have strong feelings about that v the business/developer distinction but maybe some of the reviewers can chime in.

File: docs/accordproject-faq.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+* Structured as machine readable data objects; and
+* Executed on, or integrated with, external systems (e.g. to initiate a payment or update an invoice)
+
+While the Accord Project technology is targeted at the development of smart legal contracts, the open source codebase may also be used to develop other forms of machine-readable and executable documentation.
+
+### How is an Accord Project "Smart Contract" different from "Smart Contracts" on the blockchain?
+
+Smart legal contracts should not be confused with so-called “smart contracts”, which are scripts that necessarily operates on a blockchain system. Smart contracts may form part of a smart legal contract or smart document but do not represent the agreement in its entirety. For example, a smart legal contract may include an on-chain smart contract script that is triggered to transfer a digital asset between counter-parties. Equally, a smart contract may have nothing to do with a legally binding agreement. For example, a smart legal contract may not use a blockchain system at all, such as where an agreement triggers a Stripe payment.
```
[the remark this answers is not in the document]
Comment (peterhunn) [reply docs/accordproject-faq.md]: `but may not represent the agreement in its entirety`

`A smart legal contract may not use a blockchain system at all, such as where an agreement performs an action via a web-based payment gateway.`

File: docs/accordproject-faq.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+### What benefits do Smart Legal Contracts provide?
+
+Contractual agreements sit at the heart of any organization, governing relationships with employees, shareholders, customers, suppliers, financiers, and more. Yet contracts today are not capable of being managed as the valuable assets they are. Currently, a contract exists as a static text documents stored in cloud storage services, dated contract management systems, or even email inboxes. Often these documents are Word files or PDFs that can only be interpreted by humans. A smart legal contract, by contrast, can be interpreted by machines.
```
[the remark this answers is not in the document]
Comment (peterhunn) [reply docs/accordproject-faq.md:24]: `Currently, contracts exist`

File: docs/accordproject-faq.md (hunk 1 above)
Comment (peterhunn) [reply docs/accordproject-faq.md]: Maybe remove `focused on transactions: supply chain, financial services, intellectual property, venture and token sales, real estate and construction; and also on dispute resolution.`?

File: docs/accordproject-faq.md (hunk 3 above)
Comment (peterhunn) [reply docs/accordproject-faq.md]: Remove `smart clause1. Simply `smart templates`?

File: docs/accordproject-faq.md (hunk 5 above)
Comment (peterhunn) [reply docs/accordproject-faq.md]: `You can immediately start contributing smart legal contract templates and models by using the Accord Project Template Studio.` @irmerk?

**Comment by peterhunn** [comment]
Great work @tbrooke! Many thanks for contributing.  I added a few comments principally in response to @irmerk's comments.

File: docs/accordproject-faq.md (hunk 1 above)
Comment (tbrooke) [reply docs/accordproject-faq.md]: Agreed I haven't seen much traction with those workgroups, at least on Slack. It maybe better to say something like join Slack or contact us for whatever your contractual use case is

Comment (peterhunn) [reply docs/accordproject-faq.md]: We elected to focus on core tech to begin with. The others will likely come in time.

File: docs/accordproject-developers.md [hunk 8]
```diff
@@ -5,20 +5,8 @@ title: For Developers
 
 ## Why is the Accord Project relevant for developers?
 
-The Accord Project provides a universal format for smart legal contracts, and this format is embodied in a variety of open source projects that comprise the Accord Project technology stack. Input from developers are crucial for the Accord Project.
+The Accord Project provides a universal format for smart legal contracts, and this format is embodied in a variety of open source projects that comprise the Accord Project technology stack. The Accord Project is an open source project and welcomes contributions from anyone.
 
-Developers can contribute by converting legal text into corresponding computer code, creating Accord Project templates to be used by lawyers and businesses. In addition, developers can provide input on the development of its technology stack: language, models, templating, and other tools.
+The Accord Project is developing tools including VS-Code plugins, React Components and command line tools for working with Accord Project Contracts. You can integrate contracts into existing applications, create new applications or simply assist lawyers with developing applications with the Ergo language.
```
[the remark this answers is not in the document]
Comment (tbrooke) [reply docs/accordproject-developers.md]: done

File: docs/accordproject-faq.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+The data model is used to ensure that all of the necessary data is present in the contract, and that this data is valid. In addition, it provides the necessary structure to enable contracts to "come alive" by adding executable logic.
+
+The result is a contract that is:
+
+
+* Searchable —
+* Analyzable —
+* Real-time —
+* Integrated —
```
[the remark this answers is not in the document]
Comment (tbrooke) [reply docs/accordproject-faq.md]: done

File: docs/accordproject-faq.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (tbrooke) [reply docs/accordproject-faq.md]: done

File: docs/accordproject-faq.md (hunk 1 above)
Comment (tbrooke) [reply docs/accordproject-faq.md]: changed

File: docs/accordproject-faq.md (hunk 2 above)
Comment (tbrooke) [reply docs/accordproject-faq.md]: done

File: docs/accordproject-faq.md (hunk 3 above)
Comment (tbrooke) [reply docs/accordproject-faq.md]: done

File: docs/accordproject-faq.md (hunk 4 above)
Comment (tbrooke) [reply docs/accordproject-faq.md]: done

File: docs/accordproject-faq.md (hunk 5 above)
Comment (tbrooke) [reply docs/accordproject-faq.md]: done

**Comment by dselman** [comment]
LGTM. @peterhunn should we be referring to *The* Accord Project? My understanding was that we should just use "Accord Project".

**Comment by irmerk** [comment]
Let me know if I can help with [this part](https://github.com/accordproject/techdocs/pull/302#pullrequestreview-456595171)

**Comment by tbrooke** [comment]
Yes I was confused in that it looks like a rewrite but the whole section still seems a little awkward.

**Comment by jeromesimeon** [comment]
This looks like a great contribution to me. Simply just having that FAQ section in the documentation is a big deal. I'm wondering if we can merge and fold improvements in follow up PRs, or if are there some parts that still really needs to be fixed.

Guidance please?

**Comment by irmerk** [comment]
@tbrooke I think [this guide](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) would be the most helpful in just renaming the commits and then force pushing. @jeromesimeon or myself could pair with you if you want.

@jeromesimeon I think this is a good first step and it can continue to be worked on.

**Comment by tbrooke** [comment]
Thanks, that would be great to pair to get commits down - I hope to do more and to me that is as important as the content - things like size of commits, naming, when to do a branch -- Either you or Jerome - it doesn't matter - I can do sometime tomorrow other than between 12 and 2 - Thursday and Friday are bad but next week is good -- just suggest a time

**Pull request merged**